bot/exts/evergreen/wikipedia.py

```diff
@@ -46,37 +47,39 @@ async def wiki_request(self, channel: TextChannel, search: str) -> Optional[List
         """Search wikipedia search string and return formatted first 10 pages found."""
         params = WIKI_PARAMS | {"srlimit": 10, "srsearch": search}
         async with self.bot.http_session.get(url=SEARCH_API, params=params) as resp:
-            if resp.status == 200:
-                raw_data = await resp.json()
-                number_of_results = raw_data["query"]["searchinfo"]["totalhits"]
-
-                if number_of_results:
-                    results = raw_data["query"]["search"]
-                    lines = []
-
-                    for article in results:
-                        line = WIKI_SEARCH_RESULT.format(
-                            name=article["title"],
-                            description=unescape(
-                                re.sub(
-                                    WIKI_SNIPPET_REGEX, "", article["snippet"]
-                                )
-                            ),
-                            url=f"https://en.wikipedia.org/?curid={article['pageid']}"
-                        )
-                        lines.append(line)
-
-                    return lines
-
-                else:
-                    await channel.send(
-                        "Sorry, we could not find a wikipedia article using that search term."
+            if (status := resp.status) != 200:
+                log.info(f"Unexpected response `{status}` while searching wikipedia for `{search}`")
+                raise ExternalAPIError("Wikipedia API", status)
+
+            raw_data = await resp.json()
+            if not raw_data.get("query"):
```

Comment:
```suggestion
            raw_data = await resp.json()
      
            if not raw_data.get("query"):
```
An empty line to keep things spaced.